hadoop-hdds/interface-client/src/main/proto/proto.lock

```diff
@@ -1935,4 +1935,4 @@
       }
     }
   ]
-}
+}
```

Comment:
This is a generated file by proto-backwards-compatibility.
This is a related change.